Makefile

```diff
@@ -12,38 +12,58 @@ base-image :
 	docker build -t pangeo/pangeo-notebook:master . ; \
 	docker run -w $(TESTDIR) -v $(PWD):$(TESTDIR) pangeo/pangeo-notebook:master ./run_tests.sh pangeo-notebook
 
 .PHONY: ml-notebook
 ml-notebook : base-image
 	cd ml-notebook ; \
 	conda-lock lock --mamba -k explicit -f environment.yml -f ../pangeo-notebook/environment.yml -f ../base-notebook/environment.yml  -p linux-64; \
-	../generate-packages-list.py conda-linux-64.lock > packages.txt; \
+	../generate-packages-list.py conda-linux-64.lock > packages-64.txt; \
+	conda-lock lock --mamba -k explicit -f environment.yml -f ../pangeo-notebook/environment.yml -f ../base-notebook/environment.yml  -p linux-aarch64; \
+	../generate-packages-list.py conda-linux-aarch64.lock > packages-aarch64.txt; \
+	conda-lock lock --mamba -k explicit -f environment.yml -f ../pangeo-notebook/environment.yml -f ../base-notebook/environment.yml  -p linux-ppc64le; \
+	../generate-packages-list.py conda-linux-ppc64le.lock > packages-ppc64le.txt; \
 	docker build -t pangeo/ml-notebook:master . ; \
 	docker run -w $(TESTDIR) -v $(PWD):$(TESTDIR) pangeo/ml-notebook:master ./run_tests.sh ml-notebook
 
 .PHONY: pytorch-notebook
 pytorch-notebook : base-image
 	cd pytorch-notebook ; \
 	conda-lock lock --mamba -k explicit -f environment.yml -f ../pangeo-notebook/environment.yml -f ../base-notebook/environment.yml -p linux-64; \
-	../generate-packages-list.py conda-linux-64.lock > packages.txt; \
+	../generate-packages-list.py conda-linux-64.lock > packages-64.txt; \
+	conda-lock lock --mamba -k explicit -f environment.yml -f ../pangeo-notebook/environment.yml -f ../base-notebook/environment.yml -p linux-aarch64; \
+	../generate-packages-list.py conda-linux-aarch64.lock > packages-aarch64.txt; \
+	conda-lock lock --mamba -k explicit -f environment.yml -f ../pangeo-notebook/environment.yml -f ../base-notebook/environment.yml -p linux-ppc64le; \
+	../generate-packages-list.py conda-linux-ppc64le.lock > packages-ppc64le.txt; \
 	docker build -t pangeo/pytorch-notebook:master . ; \
 	docker run -w $(TESTDIR) -v $(PWD):$(TESTDIR) pangeo/pytorch-notebook:master ./run_tests.sh pytorch-notebook
 
 .PHONY: forge
 forge : base-image
 	cd forge; \
 	conda-lock lock --mamba -k explicit -f environment.yml -f ../pangeo-notebook/environment.yml -f ../base-notebook/environment.yml -p linux-64; \
-	../generate-packages-list.py conda-linux-64.lock > packages.txt; \
+	../generate-packages-list.py conda-linux-64.lock > packages-64.txt; \
+	conda-lock lock --mamba -k explicit -f environment.yml -f ../pangeo-notebook/environment.yml -f ../base-notebook/environment.yml -p linux-aarch64; \
+	../generate-packages-list.py conda-linux-aarch64.lock > packages-aarch64.txt; \
+	conda-lock lock --mamba -k explicit -f environment.yml -f ../pangeo-notebook/environment.yml -f ../base-notebook/environment.yml -p linux-ppc64le; \
+	../generate-packages-list.py conda-linux-ppc64le.lock > packages-ppc64le.txt; \
```

Comment:
TBH I have rarely used this Makefile and am always just building things via GitHub Actions ;) I've put in a little more time to embracing `docker buildx` and think it might be the right tool to evolve this repo. in particular the `bake` tool for several related images https://github.com/scottyhq/pangeo-buildx#usage

Reply:
@scottyhq I completely agree - I love *that* repo and think that's what this should be :D What do you think needs to happen to complete that transition?

Reply:
Awesome! Mainly just more time :) I abandoned it for a bit with the "dont fix what isn't broken" rationale :) but it's also nice to keep evolving so I'll try to open a PR here this week and would love your review